docs/spicedb-dedicated/fgam.md

```diff
@@ -0,0 +1,141 @@
+
+- Service Accounts represent your workloads (e.g. your microservice issuing API requests against SpiceDB)
+- Roles grant access to the API method you want and may have optional dynamic conditions written as [CEL language expressions](https://github.com/google/cel-spec)
+- Policies bind a Service Account and a Role
+
+## Concepts
+
+### Service Accounts
+
+You'll typically want to let SpiceDB know about the identity of the services that will talk to its API. We do this with **Service Accounts**. ervice Accounts are an abstraction representing your workloads. It's not yet supported to represent Authzed console users.
+
+You'll typically create a Service Account for each service that will issue API requests against a permission system.
+
+### Roles
+
+A role represents a reusable list of permissions that can be granted to a Service Account. The permissions available represent each one of [SpiceDB's v1 API](https://buf.build/authzed/api/docs/main:authzed.api.v1):
+
+- [authzed.api.v1.PermissionsService.ReadRelationships](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.ReadRelationships)
+- [authzed.api.v1.PermissionsService.WriteRelationships](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.WriteRelationships)
+- [authzed.api.v1.PermissionsService.DeleteRelationships](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.DeleteRelationships)
+- [authzed.api.v1.PermissionsService.CheckPermission](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.CheckPermission)
+- [authzed.api.v1.PermissionsService.ExpandPermissionTree](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.ExpandPermissionTree)
+- [authzed.api.v1.PermissionsService.LookupResources](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.LookupResources)
+- [authzed.api.v1.PermissionsService.LookupSubjects](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.PermissionsService.LookupSubjects)
+- [authzed.api.v1.SchemaService.ReadSchema](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.SchemaService.ReadSchema)
+- [authzed.api.v1.SchemaService.WriteSchema](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.SchemaService.WriteSchema)
+- [authzed.api.v1.WatchService.Watch](https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WatchService.Watch)
+
+Adding the permission to a Role means each Service Account assigned the Role will have the corresponding API access. Calling the API method won't be authorized unless a Service Account has been granted explicit access via a Role and a Policy binding it.
+
+### Policy
```

Comment:
I understand the concern, and I acknowledge the confusion. I think we still have time to rename it if we think it's important.

The reason for this to be named policy is because in the future we will potentially add conditions to the policy, just like we add conditions to the permissions. But I also don't think it's a great loss if we call it role binding but still add conditions to it.

Let me know! I can go back and start adjusting the implementation 😄